README.md

```diff
@@ -35,13 +35,27 @@ brew services start postgresql
 # create the needed environemnt
 conda create -n venv_script python=3.7
 conda activate venv_script
+
 # install the backend dependencies
 cd webserver
 pip install -r requirements.txt
+
 # other parts of the project requires other dependencies....
 # instead of going through them and figuring out that you need those dependencies
 # when your code breaks, just install them now:
-pip install celery pandas cvxpy sklearn
+pip install celery==4.4.7
+pip install pandas cvxpy sklearn matplotlib s3fs flower redis xlrd

```

Comment:
Flower is a good approach!
We don't need to worry too much about this right now, but using flower complicates deployment a bit. Worth having a conversation about this later.